docs/src/prototypes/chatMessageWithPopover/ChatMessageWithPopover.tsx

```diff
@@ -0,0 +1,56 @@
+import { Chat } from '@stardust-ui/react'
+
+import * as React from 'react'
+import Popover, { handleBlur, handleFocus } from './Popover'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+class ChatMessageWithPopover extends React.Component {
+  render() {
+    return (
+      <Chat.Message
+        styles={{
+          position: 'relative',
+
+          '&.focused .actions': {
+            opacity: 1,
+          },
+          ':hover .actions': {
+            opacity: 1,
+          },
+          '& a': {
+            color: '#6264A7',
+          },
+        }}
+        author="Jane Doe"
+        timestamp="Yesterday, 10:15 PM"
+        content={{
+          content: (
+            <div>
+              <Popover className="actions" />
+              <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
+            </div>
+          ),
+        }}
+        avatar={janeAvatar}
+        onFocus={handleFocus}
+        onBlur={handleBlur}
```

Comment:
Why we're using event handlers from a separate file? It looks weird.

---

Can we use them there as class methods and just pass down to `Popover` if there is a real need?

Reply:
will change that